src/components/shared/form_fields/AmountField.vue

```diff
@@ -108,6 +109,13 @@ const setCustomAmount = ( e: Event ): void => {
 	}
 };
 
+const resetErrorInput = ( e: Event ): void => {
+	e.preventDefault();
```

Comment:
You could leave out the parameter and e.prevenDefault by using the `@focus.prevent="resetErrorInput"`, see https://vuejs.org/guide/essentials/event-handling.html#event-modifiers
But that's just a nitpick